src/fstalign.cpp

```diff
@@ -596,6 +596,9 @@ void write_stitches_to_nlp(vector<Stitching>& stitches, ofstream &output_nlp_fil
       }
 
       ref_tk = original_nlp_token;
+    } else if (stitch.comment.find("ins") == 0) {
+      logger->info("an insertion was found for {} {}", stitch.hyptk, stitch.comment);
+      ref_tk = stitch.hyptk;
```

Comment:
shouldn't we put "<ins>" in the reference token position ?

Reply:
Problem is stitch.hyptk is not printed later. What format do you propose?

Reply:
What about empty string in the ref and "ins(token)" in the stitch.comment/ali_comment?

Reply:
Well this may work too

Reply:
```
in||0.0000|0.0000|||LC|[]|[]|00:00:08.710|00:00:08.830||
||0.0000|0.0000|||||[]|||ins(let)|
lemme||0.0000|0.0000|||LC|[]|[]|00:00:09.000|00:00:09.350|sub(me)|
```